src/main/resources/application.properties

```diff
@@ -14,6 +14,8 @@ spring.jpa.hibernate.ddl-auto=update
 spring.datasource.driver-class-name=org.postgresql.Driver
 
 # Set Cache SIZE
-cache.size=5
+cache.size=100
```

Comment:
why 100? please pick this value from the cache implementation and configure its size based on this config value.

Reply:
fixed